### PR TITLE
Hit counts polish

### PR DESCRIPTION
**Pull request by jasonLaster**
* fixes the infinity bug
* re-adds the tooltip

### CODE REVIEW

File: src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx [hunk 1]
```diff
@@ -90,12 +90,15 @@ function LineHitCounts({ editor, isCollapsed, setIsCollapsed }: Props) {
         let className = styles.HitsBadge0;
         let index = null;
         if (hitCount > 0) {
```
Comment (bvaughn) [inline src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx:92]: Oh, that's a fun edge case. Should index be `NUM_GRADIENT_COLORS - 1` in this case?

File: src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ type Props = {
 
 export default function LineNumberTooltipWrapper(props: Props) {
   const { value } = useFeature("hitCounts");
-  return value ? null : <LineNumberTooltip {...props} />;
+  return (false && value) ? null : <LineNumberTooltip {...props} />;
```
Comment (bvaughn) [inline src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx:63]: I don't get it.

File: src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ export default function StaticTooltip({ targetNode, children }: StaticTooltipPro
       className={classNames(
         "pointer-events-none absolute bottom-4 z-50 mb-0.5 flex translate-x-full transform flex-row space-x-px",
         enableLargeText && "bottom-6",
-        hitCounts ? "-right-9" : "-right-1"
+        hitCounts ? "-right-[20px]" : "-right-1"
```
Comment (bvaughn) [inline src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx:21]: 😢  lol Tailwind

File: src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx (hunk 2 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx:63]: oh that wasnt supposed to be there. meant to just remove the feature flag.

File: src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx (hunk 3 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx:21]: yep

Comment (bvaughn) [inline src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx:21]: TBH I think this value would need to be dynamic too, no? B'c the gutter width depends on the number of chars.

**Pull request merged**